#
# Clicking on "Notify Offline" in Settings > Advanced > Text Events
# causes a crash. This patch fixes it.
#
--- xchat-2.4.5/src/common/text.c	29 Jul 2005 05:36:33 -0000	1.66
+++ xchat-2.4.5p1/src/common/text.c	11 Sep 2005 09:58:24 -0000
@@ -726,6 +726,7 @@
 
 static char * const pevt_generic_nick_help[] = {
 	N_("Nickname"),
+	N_("Server Name")
 };
 
 static char * const pevt_chanmodes_help[] = {
